### ISSUE TITLE
darwin - otherwise empty output when filtering for '%Disabled%' path in browser_plugins output

### ISSUE THREAD

**Issue by arubdesu** [facebook/osquery#1977]
Running 1.7.2, I noticed this anomalous-looking output (that would probably be filtered out in practice, but still doesn't seem like it should return mostly empty criteria): 
```
osquery> select * from browser_plugins where path like '%Disabled%';
+-----+------+------------+---------+-----+-------------+--------------------+--------+-----------------------------------------------+
| uid | name | identifier | version | sdk | description | development_region | native | path                                          |
+-----+------+------------+---------+-----+-------------+--------------------+--------+-----------------------------------------------+
| 0   |      |            |         |     |             |                    | 0      | /Library/Internet Plug-Ins/Disabled Plug-Ins/ |
+-----+------+------------+---------+-----+-------------+--------------------+--------+-----------------------------------------------+
```

FWIW, the contents of that dir on my 10.11.4 system is:
```
$ ls -lh /Library/Internet\ Plug-Ins/Disabled\ Plug-Ins/
total 0
drwxr-xr-x  3 root  wheel   102B Aug 22  2015 QuickTime Plugin.plugin
-rw-r--r--  1 root  wheel   2.3K Aug 22  2015 nsIQTScriptablePlugin.xpt
```

Not sure if a regression, definitely low-priority, but logging just for completeness.

**Comment by theopolis** [comment]
Right, we need to skip calling this directory a plugin, and instead walk it to discovery the plugins there-within. @PoppySeedPlehzr would you mind looking into this if you have a moment?

**Comment by theopolis** [comment]
Fixed in #1981
